packages/augur-ui/src/modules/contracts/actions/contractCalls.ts

```diff
@@ -267,15 +274,15 @@ export function createMarketRetry(market: CreateMarketData) {
     designatedReporterAddress: market.txParams._designatedReporterAddress,
     minPrice: market.txParams._prices && market.txParams._prices[0],
     maxPrice: market.txParams._prices && market.txParams._prices[1],
-    tickSize: market.txParams._numTicks,
+    numTicks: market.txParams._numTicks,
```

Comment:
don't you need to change the other createMarket call in submit-new-market to send numTicks instead of tickSize?

Reply:
it works